R/variables.R

```diff
@@ -76,12 +76,15 @@ get_age_bins <- function(groups) {
 #' @description Create individual variables for continuous and discrete age
 #'
 #' @param pop population list
-#' @param max_age maximum age - default 100
+#' @param parameters model parameters
```

Comment:
So you have removed the max_age parameter. I thought this would be useful to have this as changeable.

Reply:
As you can see on line 85, max_age becomes another model parameter. It's still parameterisable along with all the other parameters in get_parameters.

Reply:
ok